.github/BOTMETA.yml

```diff
@@ -306,6 +306,7 @@ files:
   $modules/source_control/git.py: $team_ansible
   $modules/source_control/github_key.py:
     ignored: erydo
+  $modules/storage/hpe3par/: farhan7500 gautamphegde
   $modules/storage/infinidat/: GR360RY vmalloc
```

Comment:
Add entries in `BOTMETA.yml` for:
* `$module_utils/hpe3par.py` (or `$module_utils/storage/hpe3par` see my comment below)
* `test/units/modules/storage/hpe3par`
* `lib/ansible/utils/module_docs_fragments/hpe3par`

```
$module_utils/hpe3par.py: &hpe3par
  maintainers: farhan7500 gautamphegde
  support: community
lib/ansible/utils/module_docs_fragments/hpe3par: *hpe3par
test/units/modules/storage/hpe3par: *hpe3par
```